src/rest_framework_jwt/blacklist/permissions.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.auth import get_user_model
+
+from rest_framework.permissions import IsAuthenticated
+
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+from rest_framework_jwt.utils import check_user
+
+
+class CanBlacklist(IsAuthenticated):
```

Comment:
Where is the `IsAuthenticated` check applied? If I'm reading this correctly, the current state of things will raise an `AttributeError` on [line 16](https://github.com/Styria-Digital/django-rest-framework-jwt/pull/27/files#diff-e17925b4119cfea3a2bf30816c188800R16) if the user is anonymous. This results in a 500 response to the client, not 401.

A test case for anonymous users would also help. :slightly_smiling_face: